jenkins/ci.opensuse.org/openstack-cleanvm.yaml

```diff
@@ -57,7 +61,7 @@
           function exec_jtsync {
             # only enable jtsync when build is not manually triggered
             if [[ ${ROOT_BUILD_CAUSE} != "MANUALTRIGGER" ]]; then
-              $jtsync -ci opensuse --matrix $1,$2,$3 $4 || :
+              $jtsync_fix "$jtsync -ci opensuse --matrix $1,$2,$3 $4 || :"
```

Comment:
maybe leave the `|| :` outside of the " so that a failed ssh call will not fail the run

Reply:
You are right. I changed it.